pkg/cloud/powervs.go

```diff
@@ -42,6 +42,7 @@ const (
 	PollInterval         = 5 * time.Second
 	VolumeInUseState     = "in-use"
 	VolumeAvailableState = "available"
+	secretPath           = "/var/secrets/IBMCLOUD_API_KEY"
```

Comment:
where is this been used? I'm not sure why such mistakes aren't caught in the CI.. 

created https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/issues/751 to track